app/assets/javascripts/controllers/service/service_form_controller.js

```diff
@@ -0,0 +1,52 @@
+ManageIQ.angular.app.controller('serviceFormController', ['$http', '$scope', 'serviceFormId', 'miqService', function($http, $scope, serviceFormId, miqService) {
+    var init = function() {
+
+      $scope.serviceModel = {
+        name: '',
+        description: ''
+      };
+      $scope.formId = serviceFormId;
+      $scope.afterGet = false;
+      $scope.newRecord = false;
+      $scope.modelCopy = angular.copy( $scope.serviceModel );
+      ManageIQ.angular.scope = $scope;
+
+      miqService.sparkleOn();
+      $http.get('/service/service_form_fields/' + serviceFormId).success(function(data) {
+        $scope.serviceModel.name        = data.name;
+        $scope.serviceModel.description = data.description;
+
+        $scope.afterGet = true;
+        $scope.modelCopy = angular.copy( $scope.serviceModel );
+        $scope.model = "serviceModel";
```

Comment:
Ok, sorry, one more thing.. could you move `$scope.model=` a scope up please? (As in, in `init` but not in `success`.) I'm concerned that since the success handler fires only after the request, any directives that depend on `$scope.model` being present will fail before that happens. (And also for consistency with controllers which seem to set it there.)